### ISSUE TITLE
Poker: Hitting up down arrow keys should cycle through poker cards

### ISSUE THREAD

**Issue by mattkrick** [ParabolInc/parabol#6350]
i'm lazy & i don't wanna use my mouse. i wanna just hit up/down arrow keys to select my poker card & then hit the right arrow to go to the next stage

AC
- [ ] when up arrow is tapped, the card to the right is selected
- [ ] if no card is selected, pressing up goes to the left-most card
- [ ] if the right-most card is selected, up goes to the left-most card
- [ ] the opposite happens for pressing the down arrow

Estimate: 3 hours

**Comment by Dschoordsch** [comment]
Up and down keys are not very intuitive. Left and right keys would be the right choice if not already taken.
I don't find the right key for next intuitive either. I propose reworking the key mapping (maybe check for 30 days first how much right key is used for going to next). There are also some mappings missing:
* how do you confirm your selection?
* how do you change dimension?
* how do you scroll the issue shown?

**Comment by mattkrick** [comment]
> how do you confirm your selection?

pressing up would both select & confirm all at once

> how do you change dimension?

today the right arrow moves forward 1 stage (ie moves to the next dimension, or to the next item, if on the last dimension for that item). In the other meetings the right arrow does the same thing, so that continuity is nice

> how do you scroll the issue shown?

we'd make the header card focusable, then if it's in focus it scrolls the issue. if nothing has focus, then it cycles through the poker cards

**Comment by jordanh** [comment]
Rather than let this one go, I am going to assign it to gitstart and let's see what happens!
